manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -244,9 +244,6 @@ spec:
           httpGet:
             port: 15020 # TODO: use 15021, but it spams logs
             path: /healthz/ready
-        args:
```

Comment:
Do we want to make it so we can swap the image with envoy for now? So maybe keep this, I think ztunnel ignores the args?

Reply:
Switched back to envoy as default. We can change it once we have a PR that passes all our ITs.